test/integration/targets/eos_eapi/tests/cli/zzz_reset.1

```diff
@@ -1,8 +1,9 @@
 - debug: msg="START CLI/RESET.YAML"
```

Comment:
Lets chat on IRC about this test and badtransport.1.  I dont think the test cases are valid tests but would like to discuss before simply removing them.  To answer your question, yes, changing from `.yaml` to `.<anything else>` will stop it from running